pkg/kubectl/cmd/get/get.go

```diff
@@ -210,6 +210,12 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
+	// For kubectl get events Added default sorting by lastTimestamp
+	if sortBy == "" {
```

Comment:
We should check if the flag is set, and only default it if unset, that way it is possible to manually specify the old behavior.

Reply:
done.